include/targets/generic/mrg32k3a.h

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
@jcosborn, I'm curious: have you tried using this with the CUDA target?  I imagine you'd just need to delete the `include/cuda/random_helper.h` file.

Reply:
I checked that it compiles and passes tests.  I didn't check if it produces the same numbers, but it should.

Reply:
Actually, I didn't test that correctly (I just moved the source file, without reconfiguring).  With a clean build it fails with nvcc (note #2703-D: cannot call non-constexpr function "quda::target::rng::Arr<E, N>::Arr() [with E=quda::target::rng::Trans, N=190]).  Explicitly adding a constexpr constructor (constexpr Arr() {};) doesn't help.  The original code works fine with dpcpp (which is clang-based), so I'm not sure what the problem is.

Reply:
I have got this working with `nvcc`.  Partly because `nvcc` is more exacting about `constexpr` conformance, and partly because of an undefined use case.  I'll push some fixes.